### DISCUSSION TITLE
How to select ARDS patients from MIMIC-III?

### DISCUSSION THREAD

**Discussion by lyang-xu** [MIT-LCP/mimic-code#1151] [MIMIC-III]
### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

Description of the issue, including:
* what you have tried
* references to similar issues
* queries demonstrating your question (if applicable)

**Comment by SichengH** [comment]
First, I would suggest consult with clinicians or take a look at clinal jounnals. 

Some non-clinical opinions: take a look at their reps rate and also P/F, S/F ratios.
